### PR TITLE
Remove parent from Micronaut due to conflict

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

**Comment by averikitsch** [comment]
Keeping issue #1649 open to fix conflict.

File: appengine-java11/micronaut-helloworld/pom.xml [hunk 1]
```diff
@@ -22,11 +22,11 @@
   The parent pom defines common style checks and testing strategies for our samples.
   Removing or replacing it should not affect the execution of the samples in anyway.
   -->
-  <parent>
+  <!-- <parent>
```
Comment (kurtisvg) [inline appengine-java11/micronaut-helloworld/pom.xml]: Can you add the relevant issue number for context?

Comment (averikitsch) [reply appengine-java11/micronaut-helloworld/pom.xml]: The Github issue? #1649

Comment (kurtisvg) [reply appengine-java11/micronaut-helloworld/pom.xml]: Yes please - just so we know why it's disabled.

**Pull request merged**